packages/core/package.json

```diff
@@ -84,6 +85,7 @@
     "playwright": "^1.39.0",
     "pvutils": "^1.1.3",
     "sequelize": "^6.37.1",
+    "whatwg-mimetype": "^4.0.0",
```

Comment:
Web Hypertext Application Technology Working Group (WHATWG) library that implements https://mimesniff.spec.whatwg.org/

Reply:
The two libraries have different purposes. 
mime-types primarily provides functions to look up MIME types based on file extensions and vice versa.
whatwg-mimetype is a parser for arbitrary mime types. I looked at mime-types spec and didnt see anything that would be helpful for the specific way we are using it, namely to parse a contentType